package.json

```diff
@@ -111,7 +110,6 @@
     "@types/node": "8.5.2",
     "@types/nopt": "3.0.29",
     "@types/q": "1.0.6",
-    "@types/qajax": "0.0.29",
```

Comment:
Fixed in #343 except for some dependency cleanup?

Reply:
Good catch!